lightly/data/_video.py

```diff
@@ -271,6 +271,9 @@ def __init__(self,
 
         self.videos = videos
         self.video_timestamps = video_timestamps
+        self._precomputed_length = sum((
```

Comment:
I would just call it `_length` the "precomputed" does not add any information.

Also, in terms of formatting, I'd prefer either
```python
self._length = sum((
    len(ts) for ts in self.video_timestamps))
```
or
```python
self._length = sum((
    len(ts) for ts in self.video_timestamps
))
```